library.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "tinyfsm",
+    "version": "0.3.1",
```

Comment:
Can we extract this from `include/tinyfsm.hpp` line 26 ?
If not, do you know of some common / portable place to define the version number?
I'd hate to maintain the version in multiple files.

the "library" (API compatibility) version would be 0.3

Also: does this have to match the git tag? then it would be `v0.3.0`

Reply:
>Can we extract this from include/tinyfsm.hpp line 26 ?

Not that I know of -- with some script that generates a `library.json` for each new release that can surely be done, but it needs to be pure text here.

>the "library" (API compatibility) version would be 0.3

But that's still expressed by the semantic version '0.3.2' (I've fixed to 0.3.2 btw since that's the latest release), through the major - minor - patch scheme. The version here should be really the full version of the lirary.